go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -299,3 +299,13 @@ func TestStoredProcedures(t *testing.T) {
 
 	enginetest.TestStoredProcedures(t, newDoltHarness(t))
 }
+
+func TestTransactions(t *testing.T) {
+	if !sqle.TransactionsEnabled {
+		sqle.TransactionsEnabled = true
+		defer func() {
+			sqle.TransactionsEnabled = false
```

Comment:
`defer` should probably restore the state rather than force a `false`.

Reply:
We only do this if it's false to begin with, so that's equivalent